components/package.json

```diff
@@ -6,6 +6,23 @@
             "main.js": "./main.js"
         }
     },
+    "imports": {
+        "globals": {
+            "Components": true
+        }
+    },
+    "upstream": {
+        "packages": {
+            "top": [
+                "../../../../_upstream/os-inception/*",
+                "../../../lib/*",
+                "/opt/services/*"
+            ]
+        }
+    },
+    "mappings": {
+        "firebug.sdk": "top/firebug.sdk"
```

Comment:
Map the firebug.sdk into the package and specify where to find it in the dev filesystem.